src/openapi2jsonschema/util.py

```diff
@@ -169,6 +169,54 @@ def get_request_and_response_body_components_from_paths(paths):
     return components
 
 
+def get_request_parameters_from_paths(paths):
+    components = {}
+    for path, path_definition in paths.items():
+        for http_method, http_method_definition in path_definition.items():
+            component = {}
+            if "parameters" in http_method_definition:
+                component["properties"] = {}
+                required = []
+                for param_value in http_method_definition["parameters"]:
+                    name = param_value["name"]
+                    component["properties"][name] = {"in": param_value["in"]}
+
+                    for key in param_value["schema"]:
+                        component["properties"][name][key] = param_value["schema"][key]
+
+                    if param_value["required"]:
+                        required.append(name)
+                if required:
+                    component["required"] = required
+            if "requestBody" in http_method_definition:
+                if "$ref" in http_method_definition["requestBody"]["content"][
+                    "application/json"
+                ]["schema"]:
+                    tmp = http_method_definition["requestBody"]["content"][
+                    "application/json"
+                ]["schema"]["$ref"]
+                    tmp = tmp.replace("#/components/schemas/", "") + ".json"
+                    component["requestBody"] = tmp
+                else:
+                    name_prefix_fmt = "paths_{:s}_{:s}_{{:s}}_".format(
+                        # Paths "/" and "/root" will conflict,
+                        # no idea how to solve this elegantly.
+                        path.lstrip("/").replace("/", "_") or "root",
+                        http_method,
+                    )
+                    name_prefix_fmt = re.sub(
+                        r"\{([^:\}]+)\}",
+                        r"_\1_",
+                        name_prefix_fmt,
+                    )
+                    prefix=name_prefix_fmt.format("request")
+                    component["requestBody"] = "{:s}{:s}".format(prefix, "json") + ".json"
```

Comment:
кажется так будет тоже самое
```suggestion
                    component["requestBody"] = prefix + "json.json"
```

Reply:
Готово.